### PR TITLE
Initial Jenkinsfile

### LINKED ISSUE #110: create CI pipeline

### PR DESCRIPTION
**Pull request by saberduck**
Fix #110

### CODE REVIEW

File: sonar-go-plugin/src/main/java/org/sonar/go/plugin/UastGeneratorWrapper.java [hunk 1]
```diff
@@ -44,20 +39,11 @@ private String extract() throws IOException {
     try (FileOutputStream destStream = new FileOutputStream(dest);
          InputStream streamOfExecutable = getClass().getClassLoader().getResourceAsStream(executable)) {
       copy(streamOfExecutable, destStream);
-      setExecutePermission(dest.toPath());
+      dest.setExecutable(true);
```
Comment (SonarTech) [inline sonar-go-plugin/src/main/java/org/sonar/go/plugin/UastGeneratorWrapper.java]: Vulnerability Vulnerability: Do something with the "boolean" value returned by "setExecutable". ([squid:S899](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS899))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.go%3Asonar-go&pullRequest=118&issues=AWIlgVjlH1UYAK0FSdS6&open=AWIlgVjlH1UYAK0FSdS6)

[ISSUE_KEY]: AWIlgVjlH1UYAK0FSdS6

Comment (SonarTech) [inline sonar-go-plugin/src/main/java/org/sonar/go/plugin/UastGeneratorWrapper.java]: Vulnerability Vulnerability: Do something with the "boolean" value returned by "setExecutable". ([squid:S899](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS899))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.go%3Asonar-go&pullRequest=118&issues=AWIlgVjlH1UYAK0FSdS6&open=AWIlgVjlH1UYAK0FSdS6)

[ISSUE_KEY]: AWIlgVjlH1UYAK0FSdS6

Comment (SonarTech) [inline sonar-go-plugin/src/main/java/org/sonar/go/plugin/UastGeneratorWrapper.java]: Vulnerability Vulnerability: Do something with the "boolean" value returned by "setExecutable". ([squid:S899](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS899))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.go%3Asonar-go&pullRequest=118&issues=AWIlgVjlH1UYAK0FSdS6&open=AWIlgVjlH1UYAK0FSdS6)

[ISSUE_KEY]: AWIlgVjlH1UYAK0FSdS6

Comment (SonarTech) [inline sonar-go-plugin/src/main/java/org/sonar/go/plugin/UastGeneratorWrapper.java]: Vulnerability Vulnerability: Do something with the "boolean" value returned by "setExecutable". ([squid:S899](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS899))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.go%3Asonar-go&pullRequest=118&issues=AWIlgVjlH1UYAK0FSdS6&open=AWIlgVjlH1UYAK0FSdS6)

[ISSUE_KEY]: AWIlgVjlH1UYAK0FSdS6

Comment (SonarTech) [inline sonar-go-plugin/src/main/java/org/sonar/go/plugin/UastGeneratorWrapper.java]: Vulnerability Vulnerability: Do something with the "boolean" value returned by "setExecutable". ([squid:S899](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS899))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.go%3Asonar-go&pullRequest=118&issues=AWIlgVjlH1UYAK0FSdS6&open=AWIlgVjlH1UYAK0FSdS6)

[ISSUE_KEY]: AWIlgVjlH1UYAK0FSdS6

Comment (SonarTech) [inline sonar-go-plugin/src/main/java/org/sonar/go/plugin/UastGeneratorWrapper.java]: Vulnerability Vulnerability: Do something with the "boolean" value returned by "setExecutable". ([squid:S899](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS899))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.go%3Asonar-go&pullRequest=118&issues=AWIlgVjlH1UYAK0FSdS6&open=AWIlgVjlH1UYAK0FSdS6)

[ISSUE_KEY]: AWIlgVjlH1UYAK0FSdS6

Comment (SonarTech) [inline sonar-go-plugin/src/main/java/org/sonar/go/plugin/UastGeneratorWrapper.java]: Vulnerability Vulnerability: Do something with the "boolean" value returned by "setExecutable". ([squid:S899](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS899))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.go%3Asonar-go&pullRequest=118&issues=AWIlgVjlH1UYAK0FSdS6&open=AWIlgVjlH1UYAK0FSdS6)

[ISSUE_KEY]: AWIlgVjlH1UYAK0FSdS6

**Review by alban-auzeill — APPROVED** [review]

**Pull request merged**